### PR TITLE
answer for boolean

### PR DESCRIPTION
**Pull request by aslan126**
i solwe this

### CODE REVIEW

**Comment by aysberk97** [comment]
terrible.

denied :D

**Comment by Cennetgogus** [comment]
PR 
Resolve the conflict

File: src/TeamAssignment.java [hunk 1]
```diff
@@ -1,29 +1,37 @@
...
 public class TeamAssignment {
+    public static void main(String[] args) {
 
-    // method to print out all values in a List<String> void printValues(List<String> list) {}
 
+        // method to print out all values in a List<String> void printValues(List<String> list) {}
 
 
-    // method to add all the numbers in a List<Integer>: int sum(List<Integer> list) {}
+        // method to add all the numbers in a List<Integer>: int sum(List<Integer> list) {}
```
Comment (Cennetgogus) [inline src/TeamAssignment.java:10]: second line has to be deleted

Comment (aysberk97) [reply src/TeamAssignment.java:10]: is that me?

File: src/TeamAssignment.java [hunk 2]
```diff
@@ -1,29 +1,37 @@
...
 
 
-    // to mmethod ultiply two int together: int multiply(int a, int b) {}
+        // method to multiply two int together: int multiply(int a, int b) {}
 
 
 
-    // method to tell if an int is even: boolean isEven(int number) {}
+        // method to tell if an int is even: boolean isEven(int number) {}
+        public static void main{
```
Comment (owolfhu1) [inline src/TeamAssignment.java:23]: this should be a method named isEven that returns a boolean.

**Review by owolfhu1 — CHANGES_REQUESTED** [review]

**Pull request closed without merging**